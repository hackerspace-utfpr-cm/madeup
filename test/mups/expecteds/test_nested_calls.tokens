1(0-4) ID: print
1(6-7) ID: pi
1(8-8) NEWLINE: \n
2(9-13) ID: print
2(15-17) ID: sin
2(19-21) ID: cos
2(23-24) ID: pi
2(25-25) NEWLINE: \n
3(26-26) END_OF_FILE: END OF FILE
