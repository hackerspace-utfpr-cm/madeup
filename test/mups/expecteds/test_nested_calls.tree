(block (call print (call-with-names pi)) (call print (call sin (call cos (call-with-names pi)))))
