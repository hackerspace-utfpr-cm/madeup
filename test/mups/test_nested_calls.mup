print pi
print sin cos pi
