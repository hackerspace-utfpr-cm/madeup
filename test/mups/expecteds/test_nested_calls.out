3.14159
0.0174262
